hw4/03-form.js

```diff
@@ -1,18 +1,26 @@
 const express = require('express');
-const app = express();
-const port = process.env.PORT || 5001;
+const app     = express();
+const port    = process.env.PORT || 5001;
 
 // Use middleware static() to serve all static files in the given folder
 app.use(express.static('public'));
 
-// Use middleware urlencoded() to parse an incoming request with a urlencoded payload and return an objectß
+
+// Use middleware urlencoded() to parse an incoming request with a urlencoded payload and return an object
 app.use(express.urlencoded({ extended: false }));
 
-// POST request
-app.post('', (req, res) => {
-  // Add your code here
+
+app.post('/submit', (req, res) => {
+  let newsletter = !!req.body['newsletter'] ? 'Yes, I would like the newsletter' : 'No, thank you';
```

Comment:
why havn't you checked the "on" value in newsletter?

Reply:
Good idea...I knew I was overcomplicating it.